src/table/operations.jl

```diff
@@ -0,0 +1,60 @@
+import Base: filter, map, reduce
+
+"""
+    map(f, d::DTableRows)
+
+Applies `f` to each row of `d`.
+
+# Examples
+```jldoctest        
+    d = DTable((a=rand(1000), b=rand(1000)), chunksize=100)
+    m = map(x -> x.a + x.b, eachrow(d))
+    fetch(m)
+```
+"""
+function map(f, d::DTableRows)
+    chunk_wrap = chunk -> map(f, eachrow(chunk))
```

Comment:
This should take `f` and `chunk` as arguments. I've found that closures like these will get specialized on `f`, which can cause excessive compilation when playing around in the REPL.